integration-cli/docker_cli_help_test.go

```diff
@@ -103,6 +103,9 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 			// Grab just the first word of each line
 			cmd = strings.Split(strings.TrimSpace(cmd), " ")[0]
 			cmds = append(cmds, cmd)
+			if cmd == "daemon" {
```

Comment:
In particular there are lots of tests we need to run on it - like --help, --bad-arg...